app/controllers/lectures_controller.rb

```diff
@@ -21,6 +21,7 @@ def index
   # GET courses/:course_id/lectures/1
   def show
     @current_user = current_user
+    @feedback = Feedback.new
```

Comment:
What is this doing here?

Reply:
Dunno, there seems to be some weird stuff going on with feedback anyhow - looking at the feedback controller e.g.

Reply:
in the feedback controller the weird stuff going on is authored by you according to git.

Reply:
Jup, found the issues - this branch is not based on the last version of the remote dev branch but on a history of merges of the remote dev branch with my local dev branch. Therefore some weird bits of code were dragged on. Looking into how to solve that as clean as possible now. Ideas welcome :D

Reply:
Just delete the weird feedback code lines. I think its just the lectures_controller and the feedbacks controller.

Reply:
yep did just that as in deleted my local dev branch, pulled the remote dev branch and compared the actual dev branch to our feature branch. Pushed commit and should be fine now